### ISSUE TITLE
Import VAFSC into vets-website and make demo form available in a publicly accessible location to support accessibility audit

### ISSUE THREAD

**Issue by taylorkaren** [department-of-veterans-affairs/va-forms-system-core#258]
## Context
- We want a first pass review on the accessibility status of the Q2 working demo model form so that we can assess how well we fulfilled our "accessible by default" goal and what specific or systemic accessibility problems need to be addressed before moving forward with additional work on the new VA Forms System Core (VAFSC).
- A couple of things we know for sure need to be checked:
    - We should be able to do some basic accessibility checks on the code side to confirm correct implementation of things like labels on all buttons, links and headers for screen reader accessibility.
    - Ensure meaningful error feedback is provided.
- What is the experience like when using assistive technology with the working demo model form?

## Task Description
To do from the 7/13/22 planning meeting:
- [x] Upgrade the version of VAFSC in vets website to support running the demo form.

## Acceptance Criteria
- [x] Fillable demo form is available.
- [x] Link to the fillable demo form has been added to the [VA Forms System Core Q2 2022 Working Demo Product Walkthrough page](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2219966487/VA+Forms+System+Core+Q2+Working+Demo+Product+Walkthrough).
- [x] Link to the fillable demo form has been added to the [VA Forms System Core Q2 2022 Working Demo Product Summary page](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2208432137/VA+Forms+System+Core+Q2+2022+Working+Demo+Product+Summary)
- [x] Link to the fillable demo form has been added to the [Demo Form Component Sources page](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2281472044/Demo+Form+Component+Sources)
- [x] Link to the fillable demo form has been added to the [Demo Form Conditional Elements page](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2281209918/Demo+Form+Conditional+Elements)
- [x] Karen has sent the URL out to the group that met in the 7/13/22 accessibility audit planning meeting
- [x] Karen has posted the link to the DS VA #pst-forms-library Slack channel.
- [x] Karen has sent the URL out to demo presentation and walkthrough session attendees to let them know it’s there.

**Comment by micahchiang** [comment]
In 7/20 mob session, @bjmccotter7192 mentioned there were some build errors when importing VAFSC into vets-website. This ticket should work on resolving those issues. @bjmccotter7192 can you drop a comment here elaborating on the issue? Thanks!

**Comment by bjmccotter7192** [comment]
Yeah so when we upgrade our package in vets-website we are getting alot of these errors:
`ERROR in ./src/platform/utilities/ui/asyncLoader.jsx 27:0-97
Module not found: Error: Can't resolve '@department-of-veterans-affairs/react-components/LoadingIndicator' in '/Users/bjmccotter/Projects/vets-website/src/platform/utilities/ui'
 @ ./src/applications/vaos/routes.jsx 3:0-66 36:15-26 48:15-26 60:15-26
 @ ./src/applications/vaos/vaos-entry.jsx 4:0-45 9:25-46`

I believe it has something to do with the dependencies between our package, vets-website, and the component-library

**Comment by bjmccotter7192** [comment]
Finished and merged into main: [Completed PR with important notes](https://github.com/department-of-veterans-affairs/vets-website/pull/21802)

**Comment by bjmccotter7192** [comment]
@taylorkaren here is the live link to our fillable demo form: [https://staging.va.gov/burial-poc-v6/](https://staging.va.gov/burial-poc-v6/)
